### ISSUE TITLE
CONFIG_STRING with '=' gets truncated

### ISSUE THREAD

**Issue by fabiorauber** [elastic/logstash-docker#30]
When you define a CONFIG_STRING environment variable, the string is truncated at the first '=' character. 
The value
``` 
environment:
   - "CONFIG_STRING=input { tcp { port => 5000 } } output { elasticsearch { hosts => elasticsearch:9200 } }"
```
gets truncated to
```
cat /usr/share/logstash/config/logstash.yml
config.string: input { tcp { port 
```
I believe this is happening because of env2yaml.go:
```
	// Merge any valid settings found in the environment.
	foundNewSettings := false
	for _, line := range os.Environ() {
		kv := strings.Split(line, "=")
```

**Comment by jarpy** [comment]
You are quite right! Thanks for finding this and sorry for the slow response.

This is fixed in d522fbbdd and a new image has been pushed for version 5.4.0.

Forgive me, but I never expected someone to pass the pipeline configuration in the environment. Please be aware that Logstash will merge the config string with the default configuration file, which currently is this:

```
input {
  beats {
    port => 5044
  }
}

output {
  stdout {
    codec => rubydebug
  }
}
```

Thus, all your events will end up on STDOUT (and in your Docker logs) if you don't disable the default config in some way. (Though perhaps you already are doing that).

For the future, I'll try to find a default config that will be sane for this use-case too.

**Comment by jarpy** [comment]
Continued in #37.
